astro_2.0/components/ProposalCardRenderer/components/DraftInfo/DraftInfoItem/DraftInfoItem.tsx

```diff
@@ -19,11 +21,18 @@ export const DraftInfoItem: FC<DraftInfoItemProps> = ({
   iconName,
   onClick,
   disabled,
+  tooltipText,
 }) => {
   const infoItem = (
     <>
       <div className={styles.count}>{count}</div>
-      <Icon name={iconName} className={styles.icon} />
+      {tooltipText ? (
+        <Tooltip placement="top" overlay={<span>Save Draft</span>}>
+          <Icon name={iconName} className={styles.icon} />
+        </Tooltip>
+      ) : (
+        <Icon name={iconName} className={styles.icon} />
```

Comment:
I would suggest to place it in a variable and reuse here and in line 31. 
E.g.
```
const iconComponent = <Icon name={iconName} className={styles.icon} />;
...
{tooltipText ? (
        <Tooltip placement="top" overlay={<span>Save Draft</span>}>
          {iconComponent}
        </Tooltip>
      ) : iconComponent
```

Reply:
Done